timesketch/lib/utils.py

```diff
@@ -181,3 +187,55 @@ def get_validated_indices(indices, sketch_indices):
     if exclude:
         indices = [index for index in indices if index not in exclude]
     return indices
+
+
+def send_email(subject, body, to_username, html=False):
```

Comment:
since the html is a bool, why not have it as use_html=False... makes it more "bool" like

Reply:
Good point, done.